pandas/tests/arrays/categorical/test_constructors.py

```diff
@@ -601,3 +602,14 @@ def test_constructor_imaginary(self):
         c1 = Categorical(values)
         tm.assert_index_equal(c1.categories, Index(values))
         tm.assert_numpy_array_equal(np.array(c1), np.array(values))
+
+    def test_constructor_string_and_tuples(self):
+        # GH 21416
+        # For NumPy <1.16, np.array(["", ("a",)]) raises
+        # ValueError: setting an array element with a sequence.
+        if _np_version_under1p16:
+            raise pytest.skip("Skipping for NumPy <1.16")
```

Comment:
Better to decorate the test method with ``@pytest.mark.skipif( _np_version_under1p16, reason="Skipping for NumPy <1.16")``.